feat: Add getPhoneCodesForWilaya

joeylnguyen opened the pull request:
# Description

- Added getPhoneCodesForWilaya method that returns an array of phone codes for a given wilaya code
- Added tests for getPhoneCodesForWilaya
- Exported method in index.js
- Updated tests for index.js
- Updated README

References #19 

# Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I checked that there's no dataset update (can be done by running `npm run update-dataset`)
- [x] `npm test` passes on my machine
- [x] `npm run lint` passes on my machine

joeylnguyen committed:
feat: add getPhoneCodesForWilaya

Added getPhoneCodesForWilaya method that returns an array of phone codes for a given wilaya code.

joeylnguyen committed:
Add documentation for getPhoneCodesForWilaya to README

joeylnguyen committed:
Export getPhoneCodesForWilaya and update test for index.js

joeylnguyen commented:
@Fcmam5 I plan on submitting a separate PR for getPhoneCodeForWilaya. Do you agree with that, or would you rather I add that feature to this PR?

Fcmam5 commented on README.md:
````diff
@@ -144,6 +144,22 @@ const { getWilayaByPhoneCode } = require('@dzcode-io/leblad');
 console.log(getWilayaByPhoneCode(34)); //will the wilaya object ({name: "Béjaïa"...})
 ```
 
+#### getPhoneCodesForWilaya(wilayaCode?: number)
````
This field is required

```suggestion
#### getPhoneCodesForWilaya(wilayaCode: number)
```

joeylnguyen replied:
Should be fixed! I noticed a '?' on a few of the other methods as well. Did you want me to clean those up as well?

Fcmam5 reviewed (commented):
Thank you @joeylnguyen ! I just have one tiny suggestion 😄 .

Fcmam5 commented:
> @Fcmam5 I plan on submitting a separate PR for getPhoneCodeForWilaya. Do you agree with that, or would you rather I add that feature to this PR?

@joeylnguyen It's fine to submit it in a separate PR 😄 , this will make review easier for us.  
Let's just mention that they're both related #19  

Happy Hacking !

joeylnguyen committed:
docs: fix typo in README

Removed the '?' following input parameter in method titles for getZipCodesForWilaya and
getPhoneCodesForWilaya

Fcmam5 reviewed (approved):
Thank you!

Fcmam5 commented:
@all-contributors add @joeylnguyen for code 😃
